Apps/Sandcastle/gallery/Polyline.html

```diff
@@ -73,7 +84,7 @@
         positions : Cesium.Cartesian3.fromDegreesArrayHeights([-75, 43, 500000,
                                                                -125, 43, 500000]),
         width : 10,
-        followSurface : false,
+        lineType : Cesium.LineType.STRAIGHT,
```

Comment:
I can think of `DIRECT`. Do you have any suggestions?

I do think `STRAIGHT` is a fair option since the enum is `LineType`, so we get Geodesic line, Rhumb line and Straight line.

Anyone else have any suggestions?

Reply:
I prefer `STRAIGHT`, why do we think it would be confusing?

Reply:
I think the enum name itself, `LineType` is open to change. As I wrote in the PR description, options I can think of are `CurveType` `PathType`.

Reply:
My only idea is something super pedantic like `NOT_CURVED_TO_ELLIPSOID_SURFACE` lol

Reply:
I don't feel super strongly, but I thought it was something Patrick pointed out during demo day

Reply:
What about `ArcType` `GEODESIC` `RHUMB` or `NONE`?

Reply:
Rhumb lines are straight only in a Mercator projection.

Reply:
I'm not against the proposed ArcType enum and values, I tried to google around and didn't see anything anyone else was already doing that we could reference.